web/src/UserAnimeStats.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './App.css';
```

Comment:
Why is `App.css` imported here? It should be `UserAnimeStats.css`.

Reply:
Clear all css :)